pants.toml

```diff
@@ -1,15 +1,21 @@
 [GLOBAL]
-pants_version = "2.10.0.dev3"
+pants_version = "2.10.0rc0"
 backend_packages = [
   "pants.backend.codegen.protobuf.python",
-  "pants.backend.experimental.codegen.thrift.apache.python",
+  "pants.backend.codegen.thrift.apache.python",
   "pants.backend.python",
 ]
 
 use_deprecated_python_macros = false
 
 [source]
-root_patterns = ["src/*"]
+# These are the default, but we list them explicitly for instruction.
```

Comment:
Why? Seems better not to have to do that? We want to demonstrate how easy Pants is to use, and unless someone reads the small print, they are going to see more config ( == more complexity) even though it's not actually necessary.